app/views/users/index.html.erb

```diff
@@ -1,14 +1,15 @@
-<div class="d-flex  align-items-center justify-content-center m-3">
+<div class="d-grid text-color  align-items-center justify-content-center m-3">
 <h1>Rails blog</h1>
 </div>
+
 <main >
 <% @users.each do |user| %> 
 <div class="row border border-1">
 <div class="col ">
-<div class="mb-1 ms-3  "><%=  image_tag (user.photo),height: 200 ,width: 150 %></div>
+<div class="mb-1 ms-3 "><%=  image_tag (user.photo),height: 200 ,width: 150 %></div>
 </div>
 <div class="col">
-     <%= link_to (user.name),user_path(user.id), class:"fs-2"%>
+     <%= link_to (user.name),user_path(user.id), class:"fs-2 badge badge-light"%>
      
         <div class="fs-4">Number of posts: <%= user.posts_counter%></div>
```

Comment:
- [ ] Your app is throwing an error due to misspelling. Kindly ensure to improve upon this aspect as informed by the previous code reviewer.

**View Error:**
![image](https://user-images.githubusercontent.com/12745474/158611601-ad72e534-70bf-406f-8e08-318a7e36fd6f.png)

Reply:
Hello, thanks for the feedback. This is my second review and the app works without errors on my computer, I cloned the project again a couple of times just to be sure. I cannot change anything because I have no errors.
Please  tell me how to proceed
Thanks

Reply:
Hi @sj1978 

Here are some more details about the error:

![image](https://user-images.githubusercontent.com/12745474/158626912-a93a7e13-e94d-447c-8624-316ede26ecc9.png)

Kindly check your code for the said misspelling. Thanks